error-monitoring/cron.yaml

```diff
@@ -0,0 +1,13 @@
+cron:
+- description: "File issues for errors in production"
+  url: /_cron/monitor?serviceType=production
+  schedule: every day at 7:00am
```

Comment:
Note: times are staggered so they don't all run at once and trigger various API limits